src/Api/Controllers/OrganizationExportController.cs

```diff
@@ -12,38 +13,53 @@ namespace Bit.Api.Controllers;
 [Authorize("Application")]
 public class OrganizationExportController : Controller
 {
+    private readonly ICurrentContext _currentContext;
     private readonly IUserService _userService;
     private readonly ICollectionService _collectionService;
     private readonly ICipherService _cipherService;
     private readonly GlobalSettings _globalSettings;
 
     public OrganizationExportController(
+        ICurrentContext currentContext,
         ICipherService cipherService,
         ICollectionService collectionService,
         IUserService userService,
         GlobalSettings globalSettings)
     {
+        _currentContext = currentContext;
         _cipherService = cipherService;
         _collectionService = collectionService;
         _userService = userService;
         _globalSettings = globalSettings;
     }
 
     [HttpGet("export")]
-    public async Task<OrganizationExportResponseModel> Export(Guid organizationId)
+    public async Task<IActionResult> Export(Guid organizationId)
     {
         var userId = _userService.GetProperUserId(User).Value;
 
         IEnumerable<Collection> orgCollections = await _collectionService.GetOrganizationCollections(organizationId);
         (IEnumerable<CipherOrganizationDetails> orgCiphers, Dictionary<Guid, IGrouping<Guid, CollectionCipher>> collectionCiphersGroupDict) = await _cipherService.GetOrganizationCiphers(userId, organizationId);
 
-        var result = new OrganizationExportResponseModel
+        // Backward compatibility with versions before 2022.11.0 that use ListResponseModel
+        if (_currentContext.ClientVersion < new Version("2022.11.0"))
```

Comment:
We need to bump this right?

Reply:
Not needed, this will be cherry-picked to `rc` and will be on version `2022.11`.